projects/public-search/src/app/document-detail/holdings/holdings.component.ts

```diff
@@ -63,8 +64,8 @@ export class HoldingsComponent implements OnInit {
       count = 0;
     }
     const linkText = (count > 1)
-      ? '{{ counter }} hidden holdings'
-      : '{{ counter }} hidden holding';
+      ? _('{{ counter }} hidden holdings')
+      : _('{{ counter }} hidden holding');
     return this._translateService.instant(linkText, { counter: count});
```

Comment:
I think this structure should not working. First, you translate the string (into the ternary instruction) ; and after you asked again to translate the string with the translateService.
If you need to translate a string with arguments, you need to use `translateService.stream(...)` (search in code, it's already used)

Reply:
This syntax works perfectly